components/table/table-wrapper.js

```diff
@@ -352,7 +356,7 @@ export class TableWrapper extends RtlMixin(LitElement) {
 				this._tableIntersectionObserver = new IntersectionObserver((entries) => {
 					entries.forEach((entry) => {
 						if (entry.isIntersecting) {
-							requestIdleCallback(() => this._applyClassNames(table));
+							this._applyClassNames(table);
```

Comment:
Decided to remove the `requestIdleCallback` here since it's already waiting for an animation frame. The intersection observer already runs really late, so this isn't causing any issues.